configurations/rolling-upgrade-artifacts.yaml

```diff
@@ -0,0 +1,6 @@
+# workloads
+stress_before_upgrade: cassandra-stress write cl=QUORUM n=10000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=200 -pop seq=1..10000000 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5
+stress_during_entire_upgrade: cassandra-stress read cl=QUORUM duration=25m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=200 -pop seq=1..10000000 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5
```

Comment:
This I suggest to split into the followings:
1. write command of n keys to a NEW keyspace ("-schema keyspace=keyspace2" (I don't know how many but should take few minutes, hopefully upgrade + rollback of 1 node)
2. read command that runs during the entire upgrade of the original keyspace (this one can be long one and to be killed when all upgrades complete)

Reply:
Changed